definitions/divorce/json/AuthorisationCaseField/AuthorisationCaseField-roles-and-permissions-resp-journey-nonprod.json

```diff
@@ -0,0 +1,58 @@
+[
+  {
+    "LiveFrom": "26/01/2021",
+    "CaseTypeID": "DIVORCE",
+    "CaseFieldID": "RespondentOrganisationPolicy",
+    "UserRole": "caseworker-caa",
```

Comment:
what is this role?

Reply:
We're still not 100% sure but we do know they need permissions for all the organisation policy fields. @madjava might know a bit more.

Reply:
Taken from - https://tools.hmcts.net/confluence/display/RCCD/Access+Control+Scope+of+Delivery?focusedCommentId=1471101476#comment-1471101476

This is an IdAM role.
_As part of the functionality to implement Assign Access to a Case, the IdAM role caseworker-caa was introduced to be applied to a system role only with specific code that applies for this role.  The reason for this is that the system user provides mediated access to Case Access Administrators through privileged access.  This idam-role specific code will be maintained in CCD as the Access Control solution has no real bearing on its behaviour._

Reply:
@jvictorcf ^